client/ayon_photoshop/plugins/create/create_flatten_image.py

```diff
@@ -145,14 +146,26 @@ def get_product_name(
 
         dynamic_data = prepare_template_data({"layer": "{layer}"})
 
+        get_product_name_kwargs = {}
+        if getattr(get_product_name, "use_entities", False):
+            get_product_name_kwargs.update({
+                "folder_entity": folder_entity,
+                "task_entity": task_entity,
+                "product_base_type": self.product_base_type,
+            })
+        else:
+            get_product_name_kwargs.update({
+                "task_name": task_name,
+                "task_type": task_type,
+            })
+
         product_name = get_product_name(
             project_name=project_name,
-            task_name=task_name,
-            task_type=task_type,
             host_name=host_name,
             product_type=self.product_type,
             variant=variant,
             dynamic_data=dynamic_data,
             project_entity=project_entity,
+            **get_product_name_kwargs
         )
```

Comment:
Instead of the above, couldn't this just be `product_name = super().get_product_name()`? It doesn't seem to be doing much different?